addons/autotest/functions/fnc_autotest.sqf

```diff
@@ -13,6 +13,9 @@ private _output = [];
 
     private _outputTest = call compile _code;
     if (_outputTest isEqualType []) then {
+        if (_output isNotEqualTo []) then {
```

Comment:
I'd probably wrap it with [`INFO()`](https://github.com/CBATeam/CBA_A3/blob/1f780aee7f291e7598e0855bc8ba38b1df5b0d67/addons/main/script_macros_common.hpp#L225) to indicate the start and end of output. You could also add a CBA setting to toggle RPT output.